rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -74,6 +74,7 @@
     public static final Integer LOCK_WHITELIST_UNKNOWN_ERROR_CODE = 0;
     public static final Integer LOCK_WHITELIST_SUCCESS_CODE = 1;
     public static final Integer FEE_PER_KB_GENERIC_ERROR_CODE = -10;
+    public static final Integer NEGATIVE_FEE_PER_KB_ERROR_CODE = -10;
```

Comment:
Is it ok to use an existing error code?

Reply:
I thought I could use FEE_PER_KB_GENERIC_ERROR_CODE, but I was no sure. If it is right to use that, I can change it.

Reply:
i think its clearer if its a different error code